src/python/tests/core/bot/webserver/http_server_test.py

```diff
@@ -34,15 +34,15 @@ def __init__(self):
     def write(self, data):
       self.contents += data
 
-  def __init__(self, path):
+  def __init__(self, path):  # pylint: disable=super-init-not-called
```

Comment:
can we just call the super init method instead?

Reply:
I don't think so. Calling super fails I think because the grandparent is not a child of `object` (ie: it's an old-style class).